docs/source/contributing/releasing.adoc

```diff
@@ -28,8 +21,38 @@ defined in your environment as _GITHUB_ACCESS_TOKEN_.
   For example, issues with a merged pull requests should be labeled as *resolution/done*.
 . Close milestone.
 
+[[auto-relase]]
+== Automated Release
+
+An automated release can be performed as:
+
+----
+$ export API_KEY=<api-key>
+$ export RELEASE_VERSION=<expected release version> # Eg : 1.0.0 (without 'v')
```

Comment:
Actually, we need manual process to be there as fallback option. We can't rely fully on CI job, as sometime CI job fail to start due to 403 error on invalid crumb . 

![screenshot from 2017-07-10 16-31-37](https://user-images.githubusercontent.com/1132451/28017058-db1cc762-6594-11e7-9fbd-414ff528fa17.png)